package.json

```diff
@@ -27,32 +27,25 @@
   },
   "homepage": "https://understrap.com",
   "dependencies": {
-    "bootstrap": "4.1.0",
-    "browser-sync": "^2.23.6",
+    "autoprefixer": "^8.6.2",
+    "bootstrap": "4.0.0",
```

Comment:
Should this be 4.1.0 to match the current master? Or maybe 4.1.1 to match BS4 latest?

Reply:
Yeah should be `"bootstrap": "4.1.0"`